src/lxc/cmd/lxc_user_nic.c

```diff
@@ -3,18 +3,19 @@
  * Copyright © 2013 Serge Hallyn <serge.hallyn@ubuntu.com>.
  * Copyright © 2013 Canonical Ltd.
  *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License version 2, as
- * published by the Free Software Foundation.
- *
- * This program is distributed in the hope that it will be useful,
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+
+ * This library is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License along
- * with this program; if not, write to the Free Software Foundation, Inc.,
- * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
```

Comment:
Please get the following people to agree with this change:
 - 2xsec
 - Andrey Mazo
 - Christian Brauner
 - Donghwa Jeong
 - Dongsheng Yang
 - Fajar A.
 - Henrik Kjölhede
 - Hajo Noerenberg
 - Josh Soref
 - Markus Elfring
 - S.Çağlar Onur
 - Serge Hallyn
 - Stéphane Graber
 - Tobin C.

Reply:
Ack for my part of this file.

Reply:
@2xsec @yangdongsheng @fajarnugraha @hn @jsoref @elfring @caglar10ur @hallyn @tcharding 

@brauner We're missing:
 - Andrey Mazo
 - Henrik Kjölhede

So those two definitely need to be reached out to by e-mail.

Reply:
Got ack from @caglar10ur in PR comment

Reply:
Got ack from @jsoref in PR comment.

Reply:
Got ack from @hn in PR comment.

Reply:
> @2xsec @yangdongsheng @fajarnugraha @hn @jsoref @elfring @caglar10ur @hallyn @tcharding

ACK from me

Reply:
Got ack from @hallyn in PR comment.

Reply:
@stgraber I'm sorry for late reply. I'll check it ASAP.

Reply:
@stgraber I agree.